docs/UserInterface.md

```diff
@@ -20,26 +22,162 @@ Welcome to the Taxonomy Development Tools User Interface Guide. This document is
 
 At the heart of the Taxonomy Development Tools is a robust internal database designed to streamline the management and curation of taxonomy-related data. Access to this database is facilitated through a user-friendly interface, with tables being a central component.
 
-To view the available tables, simply navigate to the Tables dropdown menu at the top of the interface.
+To view the available tables,  navigate to the Tables dropdown menu at the top of the interface.
+
+![](/Users/aa37/Documents/Documentation_Screenshots_TDT/table_dropdownmenu_AITT.png)
+
+
+TDT categorizes tables into two main types, **switch system tables** and **user tables**, each serving distinct purposes:
+
+**Switch system tables:**
+these tables are essential for the internal configuration of the TDT and cannot be modified by the users.
+
+- `table`: this table lists all the tables present in the TDT and it appears in the default page of the TDT
+
+<p align="center">
+    <img src="/Users/aa37/Documents/Documentation_Screenshots_TDT/table_AITT.png"/>
+</p>
+
+- `datatype` : this table shows all the datatype columns present in each table.
+
+<p align="center">
+    <img src="/Users/aa37/Documents/Documentation_Screenshots_TDT/datatype_table_AITT.png"/>
+</p>
+
+- `column`: this table contains all the columns present in each table.
+
+<p align="center">
+    <img src="/Users/aa37/Documents/Documentation_Screenshots_TDT/column_table_AITT.png"/>
+</p>
+
+- `message`: this table contains all the messages present one very row of each table.
+
+<p align="center">
+    <img src="/Users/aa37/Documents/Documentation_Screenshots_TDT/message_table_AITT.png/">
+</p>
+
+**User tables**
```

Comment:
Names of these tables are changing so we will need to update.  Some changes may be visible already if you pull the latest docker.